### ISSUE TITLE
force option in <VueLive> doesn't work on Chrome + VoiceOver

### ISSUE THREAD

**Issue by Jinjiang** [Jinjiang/vue-a11y-utils#3]
**Describe the bug**
force option in <VueLive> doesn't work on Chrome + VoiceOver

**To Reproduce**
Open https://codesandbox.io/s/w7jx205nll in Chrome and open VoiceOver
1. Input 'hello' in the textbox
2. Click 'Announce Immediately' button, will hear 'hello' from VoiceOver
3. Set the 'force' checkbox checked
4. Click 'Announce Immediately' button again, nothing announced

**Expected behavior**
Should hear 'hello' from VoiceOver again.

**Screenshots**
(no)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome + VoiceOver
 - Version: 70
